### ISSUE TITLE
contenteditable selector is inadequate, not compatible with HAML-generated HTML

### ISSUE THREAD

**Issue by costa** [NYTimes/backbone.stickit#268]
According to http://haml.info/docs/yardoc/file.REFERENCE.html#boolean_attributes the selector at https://github.com/NYTimes/backbone.stickit/blob/13a08b67a236186bb41363bd4ca49de4fadba39f/backbone.stickit.js#L431 is incorrect. Should be `'[contenteditable]'`

**Comment by costa** [comment]
Hey, it is this time of the year...
Now I would like to make a pull, however, I'm experiencing:
```
$ grunt qunit
Running "qunit:all" (qunit) task
Testing http://localhost:3000/test/index.html ...ERROR
Warning: PhantomJS unable to load "http://localhost:3000/test/index.html" URI. Use --force to continue.

Aborted due to warnings.
```
Debugging this would be nice, but it's not on my critical path at the moment. I'll appreciate if anyone shares any ideas on how to fix this on OSX 10.10 though.
Thus, I don't mind if someone else fixes that line.

**Comment by akre54** [comment]
Is `npm test` working for you?

**Comment by costa** [comment]
well kinda...
```
$ npm test

> backbone.stickit@0.8.0 test {{.}}
> phantomjs test/vendor/runner.js test/index.html?noglobals=true

Took 536ms to run 371 tests. 370 passed, 1 failed.
Unsafe JavaScript attempt to access frame with URL about:blank from frame
with URL file://test/vendor/runner.js. Domains, protocols and ports must
match.

Unsafe JavaScript attempt to access frame with URL about:blank from frame
with URL file://test/vendor/runner.js. Domains, protocols and ports must
match.

Unsafe JavaScript attempt to access frame with URL about:blank from frame
with URL file://test/vendor/runner.js. Domains, protocols and ports must
match.

npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

**Comment by akre54** [comment]
According to this [stackoverflow post](http://stackoverflow.com/a/27083831) it's a harmless bug in phantomjs, fixed in ariya/phantomjs#12720.

There does seem to be a failing test too though. I'll look into it.

**Comment by akre54** [comment]
In the meantime, feel free to ignore that message and the failing test. Looking forward to your pull!
